### ISSUE TITLE
Evaluate Performance of pipeline generated from TPOT

### ISSUE THREAD

**Issue by prasad0896** [EpistasisLab/tpot#582]
Generated Pipeline for an example makes use of Random Forest Regressor. How to evaluate the performance of this pipeline generated by TPOT alongwith the stationary available data. And also how to handle Time series using TPOT ( any time series examples making use of TPOT available?)

**Comment by rhiever** [comment]
When you say "generated pipeline," do you mean the pipeline code generated via the `export` function?

Otherwise if by "generated pipeline" you mean the "best" pipeline discovered during the optimization process, then you can simply use the `score` function and provide it with the test data and labels.

>And also how to handle Time series using TPOT ( any time series examples making use of TPOT available?)

TPOT would handle time series the same way that scikit-learn does. You can use [TimeSeriesSplit](http://scikit-learn.org/stable/modules/generated/sklearn.model_selection.TimeSeriesSplit.html) for the `cv` parameter.

**Comment by prasad0896** [comment]
Ok thanks. I will try that. Would it be possible to add a time series example in the documentation. Because afterall Time Series prediction does make use of Regressions

**Comment by rhiever** [comment]
I filed an issue suggesting to add such an example (#584). We're currently seeking contributors for the issue.

Will close this issue for now. Please feel free to re-open the issue if you have more questions.

**Comment by ikramelmbarki** [comment]
> When you say "generated pipeline," do you mean the pipeline code generated via the `export` function?
> 
> Otherwise if by "generated pipeline" you mean the "best" pipeline discovered during the optimization process, then you can simply use the `score` function and provide it with the test data and labels.
> 
> > And also how to handle Time series using TPOT ( any time series examples making use of TPOT available?)
> 
> TPOT would handle time series the same way that scikit-learn does. You can use [TimeSeriesSplit](http://scikit-learn.org/stable/modules/generated/sklearn.model_selection.TimeSeriesSplit.html) for the `cv` parameter.

there is any new solutions for multivariate time series with TPOT? please
